qiita_db/test/test_software.py

```diff
@@ -233,6 +245,14 @@ def test_add_publications(self):
                ['10.1000/nmeth.f.101', '12345678']]
         self.assertItemsEqual(tester.publications, exp)
 
+    def test_environment_script(self):
+        tester = qdb.software.Software(1)
+        self.assertEqual(tester.environment_script, 'source activate qiita')
```

Comment:
That is added in the database - In the last meeting that I attended we agreed that we are going to use conda environments to manage environment encapsulation. However, I found that I can actually leave it up to the user (so it can even put custom scripts). The test that we have in the database however, assumes a conda environment, as that is what we are going to recommend.